runtime/queries/c3/highlights.scm

```diff
@@ -0,0 +1,323 @@
+(field_expr field: (access_ident (ident) @variable.member))
+(struct_member_declaration (ident) @variable.member)
+(struct_member_declaration (identifier_list (ident) @variable.member))
+(bitstruct_member_declaration (ident) @variable.member)
+(initializer_list (arg (param_path (param_path_element (ident) @variable.member))))
+;; 2) Parameter
+(parameter name: (_) @variable.parameter)
+(call_invocation (arg (param_path (param_path_element [(ident) (ct_ident)] @variable.parameter))))
+(enum_param_declaration (ident) @variable.parameter)
+;; 3) Declaration
+(global_declaration (ident) @variable.declaration)
+(local_decl_after_type name: [(ident) (ct_ident)] @variable.declaration)
+(var_decl name: [(ident) (ct_ident)] @variable.declaration)
+(try_unwrap (ident) @variable.declaration)
+(catch_unwrap (ident) @variable.declaration)
+
+;; Keyword (from `c3c --list-keywords`)
+[
+  "assert"
+  "asm"
+  "catch"
+  "defer"
+  "try"
+  "var"
+] @keyword
+
+[
+  "$alignof"
+  "$and"
+  "$append"
+  "$assert"
+  "$assignable"
+  "$case"
+  "$concat"
+  "$default"
+  "$defined"
+  "$echo"
+  "$else"
+  "$embed"
+  "$endfor"
+  "$endforeach"
+  "$endif"
+  "$endswitch"
+  "$eval"
+  "$evaltype"
+  "$error"
+  "$exec"
+  "$extnameof"
+  "$feature"
+  "$for"
+  "$foreach"
+  "$if"
+  "$include"
+  "$is_const"
+  "$nameof"
+  "$offsetof"
+  "$or"
+  "$qnameof"
+  "$sizeof"
+  "$stringify"
+  "$switch"
+  "$typefrom"
+  "$typeof"
+  "$vacount"
+  "$vatype"
+  "$vaconst"
+  "$varef"
+  "$vaarg"
+  "$vaexpr"
+  "$vasplat"
+] @keyword.directive
+
+"fn" @keyword.function
+"macro" @keyword.function
+"return" @keyword.return
+"import" @keyword.import
+"module" @keyword.module
+
+[
+  "bitstruct"
+  "def"
+  "distinct"
+  "enum"
+  "fault"
+  "interface"
+  "struct"
+  "union"
+] @keyword.type
+
+[
+  "case"
+  "default"
+  "else"
+  "if"
+  "nextcase"
+  "switch"
+] @keyword.conditional
+
+[
+  "break"
+  "continue"
+  "do"
+  "for"
+  "foreach"
+  "foreach_r"
+  "while"
+] @keyword.repeat
+
+[
+  "const"
+  "extern"
+  "inline"
+  "static"
+  "tlocal"
+] @keyword.modifier
+
+;; Operator (from `c3c --list-operators`)
+[
+  "&"
+  "!"
+  "~"
+  "|"
+  "^"
+  ;; ":"
+  ;; ","
+  ;; ";"
+  "="
+  ">"
+  "/"
+  "."
+  ;; "#"
+  "<"
+  ;; "{"
+  ;; "["
+  ;; "("
+  "-"
+  "%"
+  "+"
+  "?"
+  ;; "}"
+  ;; "]"
+  ;; ")"
+  "*"
+  ;; "_"
+  "&&"
+  ;; "->"
+  "!!"
+  "&="
+  "|="
+  "^="
+  "/="
+  ".."
+  "?:"
+  "=="
+  ">="
+  "=>"
+  "<="
+  ;; "{|"
+  ;; "(<"
+  ;; "[<"
+  "-="
+  "--"
+  "%="
+  "*="
+  "!="
+  "||"
+  "+="
+  "++"
+  ;; "|}"
+  ;; ">)"
+  ;; ">]"
+  "??"
+  ;; "::"
+  "<<"
+  ">>"
+  "..."
+  "<<="
+  ">>="
+] @operator
+
+(range_expr ":" @operator)
+(foreach_cond ":" @operator)
+
+(ternary_expr
+  [
+    "?"
+    ":"
+  ] @keyword.conditional.ternary)
+
+(elvis_orelse_expr
+  [
+    "?:"
+    "??"
+  ] @keyword.conditional.ternary)
+
+;; Literal
+(integer_literal) @number
+(real_literal) @number.float
+(char_literal) @character
+(bytes_literal) @number
```

Comment:
Some of the scopes in this file need fixing https://docs.helix-editor.com/themes.html#syntax-highlighting

Reply:
Is there a good way to debug these scopes ?

Reply:
You can use the `:tree-sitter-highlight-name` command to show the capture name under the cursor